hack/ci/e2e.sh

```diff
@@ -97,18 +97,83 @@ build() {
 
 # up a cluster with kind
 create_cluster() {
+    # create the audit-policy necessary for API Coverage
+    # https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy
+    cp $(dirname $0)/audit-policy.yaml /tmp/audit-policy.yaml
+    touch "${ARTIFACTS}/apiserver-audit.log"
     # create the config file
     cat <<EOF > "${ARTIFACTS}/kind-config.yaml"
 # config for 1 control plane node and 2 workers
 # necessary for conformance
 kind: Cluster
 apiVersion: kind.sigs.k8s.io/v1alpha3
 nodes:
-# the control plane node
+# the control plane node / apiservers
 - role: control-plane
+  extraMounts:
+  - hostPath: /tmp/audit-policy.yaml
+    containerPath: /etc/kubernetes/audit-policy.yaml
+  - hostPath: "${ARTIFACTS}/apiserver-audit.log"
+    containerPath: /var/log/apiserver-audit.log
 - role: worker
 - role: worker
 EOF
+    # open to other ways to detect the kubeadm version
```

Comment:
kubeadm and kubectl are unavailable during this portion of the CI job... I've also looked at trying to detect via looking at the k8s source tree.
Any suggestions?

Reply:
The version of k8s is stored in the file `/kind/version` inside the nodes